internal/util/search/search.go

```diff
@@ -38,7 +40,12 @@ type SearchReplace struct {
 	// filtered by path and/or value
 	PutLiteral string
 
+	// PutPattern is the setters reference comment to be added at to field
+	PutPattern string
+
 	filePath string
+
+	PackagePath string
```

Comment:
What is the difference between `filePath` and `PackagePath`?

Reply:
Package path is path to package. filePath is the path of the file which is being parsed. It is the place holder so that the underlying methods can use it to print the filePath as output. It is private and the clients should not care about it.